ACE/ace/config-macosx-iOS-hardware.h

```diff
@@ -3,12 +3,12 @@
 
 #define ACE_HAS_IPHONE
 #define ACE_SIZEOF_LONG_DOUBLE 8
+#define ACE_LACKS_SYSTEM
 
-#include "ace/config-macosx-mavericks.h"
+#include "ace/config-macosx-mojave.h"
 
 #ifdef ACE_HAS_SYSV_IPC
 #undef ACE_HAS_SYSV_IPC
 #endif
 
 #endif /* ACE_CONFIG_MACOSX_IPHONE_HARDWARE_H */
```

Comment:
Do "simulator" and "hardware" actually need separate config files?  Seems like a simulator should be able to run the same code as the hardware.  The only differences are `ACE_SIZEOF_LONG_DOUBLE` and `ACE_HAS_SYSV_IPC`.